### PR TITLE
Update feature_request.md

### PR DESCRIPTION
**Pull request by ninavizz**
Echoing similar PR in docs repo, below (verbaitm from that PR):

> 1. "Has this idea been influenced by your conversations with users, or by your own experience as a SecureDrop user?" is not user research evidence. It's anecdotal feedback. Qualifying anecdotal feedback as user research evidence (the latter, aggregate information collected in a methodical fashion) is a disservice to the difference between the two. Anecdotal feedback is absolutely important, however—they just need to be correctly qualified.
> 2. Who the users are, needs to be more clearly spoken to in issues. Centering issues on existing user profiles (mini-personae) can be helpful.
> 3. In 3 years, I've not once produced any findings from user research to merit updates to the docs. As such, "User Research Evidence" as its own section is both moot and problematic.

And, 

4. "Security," "Product Desirability," and "Usability" are very different primary needs that SecureDrop dev efforts work in service to. Human error is often a security concern, and nefarious behavior by humans is at the root of most security concerns—however it has been my own observation in securedrop Issues, that both should be spoken-to, separately. Where they overlap, that can certainly be spoken to—but I feel it would better support both, to seek to speak to both.

## Status

Ready for review / Work in progress

## Description of Changes

Fixes #.

Changes proposed in this pull request:

## Testing

How should the reviewer test this PR?
Write out any special testing steps here.

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.

### If you made changes to the server application code:

- [ ] Linting (`make lint`) and tests (`make test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [ ] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you added or removed a file deployed with the application:

- [ ] I have updated AppArmor rules to include the change

### If you made non-trivial code changes:

- [ ] I have written a test plan and validated it for this PR

Choose one of the following:

- [ ] I have opened a PR in the [docs repo](https://github.com/freedomofpress/securedrop-docs) for these changes, or will do so later
- [ ] I would appreciate help with the documentation
- [ ] These changes do not require documentation

### If you added or updated a production code dependency:

Production code dependencies are defined in:

- `admin/requirements.in`
- `admin/requirements-ansible.in`
- `securedrop/requirements/python3/securedrop-app-code-requirements.in`

If you changed another `requirements.in` file that applies only to development
or testing environments, then no diff review is required, and you can skip
(remove) this section.

Choose one of the following:

- [ ] I have performed a diff review and pasted the contents to [the packaging wiki](https://github.com/freedomofpress/securedrop-debian-packaging/wiki)
- [ ] I would like someone else to do the diff review

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -3,15 +3,18 @@ name: Feature request
...
 
 A short summary of the idea.
 
-## User Research Evidence
+## How will this impact [SecureDrop users](https://github.com/freedomofpress/securedrop-ux/wiki/Users)?
+<!-- How do you feel this change might impact SecureDrop's usability, accessibility, or usefulness—and specifically, for which users? Has anecdotal feedback from users influenced this change? Does evidence exist from user research to support this idea? Could design or user research efforts be helpful to support a change? -->
 
-Does evidence exist from user research to support this idea? Could research or design help validate or improve it? Has this idea been influenced by your conversations with users, or by your own experience as a SecureDrop user? If so, please note it here, and [take a look at our UX wiki](https://github.com/freedomofpress/securedrop-ux/wiki) for more information about our design and research efforts.
+## How does this work in support of SecureDrop's [Threat Model](https://docs.securedrop.org/en/stable/threat_model/threat_model.html)?
+<!-- While active users do benefit from security considerations, all folks considered in the threat model are served by security-specific things. -->
```
Comment (eloquence) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I would reword this:
## How would this affect SecureDrop's [threat model](https://docs.securedrop.org/en/stable/threat_model/threat_model.html)?
Would this change create new risks for sources, journalists, or administrators? Would it mitigate existing risks?

**Comment by eloquence** [comment]
The threat model language is a bit unclear to me and I've suggested alternative wording there. Otherwise this looks like a good change from my end.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -3,15 +3,18 @@ name: Feature request
...
 
 ## User Stories
+<!-- “As a [persona], I [want to], [so that].” -->
```
Comment (eloquence) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I would just say `[role]` here instead of `[persona]`; it's not necessary to refer to specific UX personas in user stories, and the term is less helpful without further explanation. Alternatively, we could simply give a full example of a user story.

**Comment by eloquence** [comment]
I've appended a commit with the suggested changes.

**Review by cfm — APPROVED** [review]
Thanks for proposing this change, @ninavizz of yesteryear, and for tacking on your suggested revisions, @eloquence of this year!  I'm happy with this version and will approve _tentatively_ pending any input from @l3th3 re: the new threat-modeling language.  I'll merge a week from today if there are no objections.

**Comment by cfm** [comment]
Superseded by #6734, which rebases 8d9aa6e0946bb23d72a63344c8b9e02a08e42e2e.
